utils.py

```diff
@@ -1,9 +1,19 @@
 import pandas as pd
 import os
 import random
+import numpy as np
+import matplotlib.pyplot as plt
 
 
 class SIR:
+    S = []
```

Comment:
@Rohan-Kamat Can we have a more explanatory variable name instead of `S`?
Also, it is better not to start variable names with capital letter.